src/lib/components/dropdown/Dropdown.component.js

```diff
@@ -41,43 +63,19 @@ const DropdownMenuStyled = styled.ul`
   position: absolute;
   margin: 0;
   padding: 0;
-  border: 1px solid ${getThemePropSelector("primary")};
```

Comment:
Please double-check with the design, we should have a border for each of the items.

Reply:
done